torch/_torch_docs.py

```diff
@@ -993,60 +993,71 @@ def merge_dicts(*dicts):
            r"""
 asarray(obj, *, dtype=None, device=None, copy=None, requires_grad=False) -> Tensor
 
-Converts :attr:`obj` into a tensor, sharing data and preserving autograd history
-if possible.
+Converts :attr:`obj` in a tensor.
```

Comment:
nit - Converts :attr:`obj` to a tensor.

Reply:
My mistake! Sorry about that, @ysiraichi